### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by jparisu**
Implement print functions

Requirements:

- #2

ToDo:

- [x] Documentation
- [x] CI

### CODE REVIEW

File: docs/rst/exports/alias.include [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.. |FastDDSDocs| replace:: `Fast DDS Documentation <https://fast-dds.docs.eprosima.com/en/v2.10.0/>`__
+.. |DDSSpecification| replace:: `DDS specification <https://www.omg.org/spec/DDS/1.4/About-DDS/>`__
+
+.. |ddsrouter| replace:: *DDS Router*
+.. |fastdds| replace:: *Fast DDS*
+.. |spy| replace:: *Fast DDS Spy*
+
+.. |eddsrouter| replace:: *eProsima DDS Router*
+.. |efastdds| replace:: *eProsima Fast DDS*
+.. |eddsrecord| replace:: *eProsima Fast DDS Spy*
```
Comment (juanlofer-eprosima) [inline docs/rst/exports/alias.include:10]:
```suggestion
```

File: docs/rst/formalia/titlepage.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  :align: left
+  :alt: eProsima
+  :target: http://www.eprosima.com/
+
+*eProsima Fast DDS Spy* is an end-user software application that efficiently saves DDS data published in a DDS environment in a MCAP format database.
+Thus, the exact playback of the recorded network events is possible as the data is linked to the timestamp at which the original data was published.
+At the moment, it is only possible to replay the data using external tools capable of interpreting the MCAP format, as the *eProsima Fast DDS Spy* does not provide a replay tool.
+
+*eProsima Fast DDS Spy* is easily configurable and installed with a default setup, so that DDS topics, data types and entities are automatically discovered without the need to specify the types of data recorded.
+This is because the recording tool exploits the DynamicTypes functionality of `eProsima Fast DDS <https://fast-dds.docs.eprosima.com>`_, the C++ implementation of the `DDS (Data Distribution Service) Specification <https://www.omg.org/spec/DDS/About-DDS/>`_ defined by the `Object Management Group (OMG) <https://www.omg.org/>`_.
```
Comment (juanlofer-eprosima) [inline docs/rst/formalia/titlepage.rst:19]: Pending to update.

**Review by juanlofer-eprosima — CHANGES_REQUESTED** [review]

File: docs/rst/developer_manual/installation/sources/linux.rst [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+        colcon build --packages-up-to-regex fastddsspy
+
+.. note::
+
+    Being based on CMake_, it is possible to pass the CMake configuration options to the :code:`colcon build` command.
+    For more information on the specific syntax, please refer to the `CMake specific arguments <https://colcon.readthedocs.io/en/released/reference/verb/build.html#cmake-specific-arguments>`_ page of the colcon_ manual.
+
+.. TODO check if this is correctly parsed, because in local it does not render correctly
+
+CMake installation
```
Comment (juanlofer-eprosima) [inline docs/rst/developer_manual/installation/sources/linux.rst:198]: I would keep the same structure as in other repos. Also, the linked instructions are unindexed.

**Review by juanlofer-eprosima — CHANGES_REQUESTED** [review]

File: docs/rst/developer_manual/installation/sources/linux.rst (hunk 3 above)
Comment (jparisu) [reply docs/rst/developer_manual/installation/sources/linux.rst:198]: The idea was to degrade cmake option and not show it unless you are really looking for it. This project uses 9 colcon subpackages, it starts to not make much sense to use cmake to compile it.

Comment (juanlofer-eprosima) [reply docs/rst/developer_manual/installation/sources/linux.rst:198]: @rsanchez15 ?

Comment (jparisu) [reply docs/rst/developer_manual/installation/sources/linux.rst:198]: I would prefer not to. I do not like other repos documentation, it seems indirect and difficult to follow. I would leave CMake only for those experienced users that really want to look for it

**Pull request merged**